easybuild/easyconfigs/d/DeepLabCut/DeepLabCut-2.2.0.6-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,68 @@
+# Loosely based on PR #7680 
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'DeepLabCut'
+version = '2.2.0.6'
+versionsuffix = '-CUDA-%(cudaver)s'
+local_version = '%(versionsuffix)s-contrib'
+
+homepage = 'http://www.mousemotorlab.org/deeplabcut'
+description = "Markerless tracking of user-defined features with deep learning"
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'pic': True}
+
+dependencies = [
+    ('CUDA', '11.3.1', '', SYSTEM),
+    ('cuDNN', '8.2.1.32', '-CUDA-%(cudaver)s', SYSTEM),
+    ('Python', '3.9.5'),
+    ('h5py', '3.2.1'),
+    ('IPython', '7.25.0'),
+    ('scikit-learn', '0.24.2'),
+    ('scikit-image', '0.18.3'),
+    ('TensorFlow', '2.6.0', versionsuffix),
+    ('PyYAML', '5.4.1'),
+    ('PyTables', '3.6.1'),
+    ('wxPython', '4.1.1'),
+    ('OpenCV', '4.5.3', local_version),
+    ('numba', '0.53.1'),
+    ('libyaml', '0.2.5'),
+    ('imgaug', '0.4.0', versionsuffix),
+    ('tqdm', '4.61.2'),
+    ('ruamel.yaml', '0.17.21'),
+    ('PyYAML', '5.4.1'),
+    ('statsmodels', '0.12.2'),
+    ('PyTorch', '1.10.0', versionsuffix),
+]
+
+use_pip = True
+
+exts_list = [
+    ('filterpy', '1.4.5', {
+        'sources': ['%(name)s-%(version)s.zip'],
+        'checksums': ['4f2a4d39e4ea601b9ab42b2db08b5918a9538c168cff1c6895ae26646f3d73b1'],
+    }),
+    ('pandas', '1.3.3', {
```

Comment:
we bundle `pandas` in `SciPy-bundle`, is there a need for a version newer than `1.2.4`?

Reply:
Probably no need, most likely I did not know and I was thus looking for the latest version of it. I have removed it and it is still building. Thanks for pointing that out.